### DISCUSSION TITLE
Let the program use/upload to the Spam Lists repo.

### DISCUSSION THREAD

**Discussion by TechStudent10** [ThioJoe/YT-Spammer-Purge#709] [Ideas]
Another (and probably easier) method would be that when the program finds spam, it automatically commits/creates a PR on the [`ThioJoe/YT-Spam-Lists`](https://github.com/ThioJoe/YT-Spam-Lists) repo. That way the program can automatically reference these lists and use them for spam detection.

_Originally posted by @TechStudent11 in https://github.com/ThioJoe/YT-Spammer-Purge/discussions/298#discussioncomment-2260983_

**Comment by Firecul** [comment]
1) Anyone wanting to take part in that would also have to authorise the app for GitHub in addition to YT for that to work.
2) Having, potentially, hundreds of PR posted per day would be near unmanageable for just one person, or even a small team.
3) One person could, in theory, poison the spam lists so that a lot of innocent comments would be caught as a false positive. This would mean every request would need to get vetted, ruling out an automated approach to 2)

**Reply by TechStudent10** [reply]
> 1. Anyone wanting to take part in that would also have to authorise the app for GitHub in addition to YT for that to work.

It could send a POST request to a server with an encrypted authorization key to commit using the GitHub API/Apps.

> 2\. Having, potentially, hundreds of PR posted per day would be near unmanageable for just one person, or even a small team.

It could have one PR with a single branch per week/update.

> 3\. One person could, in theory, poison the spam lists so that a lot of innocent comments would be caught as a false positive. This would mean every request would need to get vetted, ruling out an automated approach to 2)

That is the only one I don't have a "solution" for.

---

Okay so my idea isn't perfect.

**Reply by Firecul** [reply]
> Okay so my idea isn't perfect.

Nothing is, I'm just better at shooting holes in things rather than coming up with good ideas myself 😞

**Reply by TechStudent10** [reply]
> > Okay so my idea isn't perfect.
> 
> Nothing is, I'm just better at shooting holes in things rather than coming up with good ideas myself 😞

That's fine. You can identify problems in ideas!

**Reply by DinhHuy2010** [reply]
oof

**Reply by TechStudent10** [reply]
> oof

@DinhHuy2010 you shouldn't be leaving seemingly random comments on posts.
